src/um/dwmapi.rs

```diff
@@ -63,32 +118,55 @@ STRUCT!{#[repr(packed)] struct DWM_TIMING_INFO {
     cPixelsDrawn: ULONGLONG,
     cBuffersEmpty: DWM_FRAME_COUNT,
 }}
-pub const DWMWA_NCRENDERING_ENABLED: DWORD = 1;
-pub const DWMWA_NCRENDERING_POLICY: DWORD = 2;
-pub const DWMWA_TRANSITIONS_FORCEDISABLED: DWORD = 2;
-pub const DWMWA_ALLOW_NCPAINT: DWORD = 4;
-pub const DWMWA_CAPTION_BUTTON_BOUNDS: DWORD = 5;
-pub const DWMWA_NONCLIENT_RTL_LAYOUT: DWORD = 6;
-pub const DWMWA_FORCE_ICONIC_REPRESENTATION: DWORD = 7;
-pub const DWMWA_FLIP3D_POLICY: DWORD = 8;
-pub const DWMWA_EXTENDED_FRAME_BOUNDS: DWORD = 9;
-pub const DWMWA_HAS_ICONIC_BITMAP: DWORD = 10;
-pub const DWMWA_DISALLOW_PEEK: DWORD = 11;
-pub const DWMWA_EXCLUDED_FROM_PEEK: DWORD = 12;
-pub const DWMWA_CLOAK: DWORD = 13;
-pub const DWMWA_CLOAKED: DWORD = 14;
-pub const DWMWA_FREEZE_REPRESENTATION: DWORD = 15;
-pub const DWMWA_LAST: DWORD = 16;
+ENUM!{enum DWM_SOURCE_FRAME_SAMPLING {
+    DWM_SOURCE_FRAME_SAMPLING_POINT = 0,
+    DWM_SOURCE_FRAME_SAMPLING_COVERAGE = 1,
+    DWM_SOURCE_FRAME_SAMPLING_LAST = 2,
+}}
+// pub const c_DwmMaxQueuedBuffers: UINT = 8;
+// pub const c_DwmMaxMonitors: UINT = 16;
+// pub const c_DwmMaxAdapters: UINT = 16;
+STRUCT!{#[repr(packed)] struct DWM_PRESENT_PARAMETERS {
+    cbSize: UINT32,
+    fQueue: BOOL,
+    cRefreshStart: DWM_FRAME_COUNT,
+    cBuffer: UINT,
+    fUseSourceRate: BOOL,
+    rateSource: UNSIGNED_RATIO,
+    cRefreshesPerFrame: UINT,
+    eSampling: DWM_SOURCE_FRAME_SAMPLING,
+}}
+// pub const DWM_FRAME_DURATION_DEFAULT: i32 = -1;
+FN!{stdcall DwmDefWindowProc(
+        hWnd: HWND,
+        msg: UINT,
+        wParam: WPARAM,
+        lParam: LPARAM,
+        plResult: *mut LRESULT,
```

Comment:
These parameters are still all excessively indented.

Reply:
Issues with indentations of everything related to macros in the IDE that I use + no help from rustfmt + my own blindness :smile: